src/sentry/utils/json.py

```diff
@@ -70,6 +70,7 @@ def dump(value, fp, **kwargs):
 
 
 def dumps(value, escape=False, **kwargs):
+    # Legacy use. Do not use. Use dumps_htmlsafe
```

Comment:
It's bad naming. It's just html safe, it does not "escape" in the traditional sense. Also yes, it was not mark_safe.